.github/workflows/test_catch_flaky.yml

```diff
@@ -0,0 +1,46 @@
+name: catch_flaky
+
+on:
+  pull_request
+
+jobs:
+  linux:
+    # We want to run on external PRs, but not on our own internal
+    # PRs as they'll be run by the push to the branch. The job is also not
+    # launched, if he 'notest' or 'noflaky' label is set.
```

Comment:
I didn't get this. Will it be running for PRs in vshard repo automatically or not? What are internal and external PRs? And why should it not run on one of them? Don't we want to detect flaky tests for all PRs (except having these special labels)?